webpack/assets/javascripts/react_app/components/ToastsList/ToastList.scss

```diff
@@ -0,0 +1,9 @@
+/* Make space for <header> component */
+.foreman-toast {
+  top: 1em;
```

Comment:
can we also push it a bit away from the nav?
before it had this amount of space:
![Screenshot - 2021-07-08T153705 184](https://user-images.githubusercontent.com/26363699/124922946-a8b62d80-e002-11eb-923d-a45367d2ab65.png)

and now it's almost touching the nav

Reply:
I actually think the toast should move it up.
Can we remove this so the user can see the action buttons?
with `top: 1em;`
![Screenshot from 2021-07-09 15-56-28](https://user-images.githubusercontent.com/30431079/125089105-be952280-e0d6-11eb-8bcf-ce19cf767294.png)
without `top: 1em;`
![Screenshot from 2021-07-09 15-56-23](https://user-images.githubusercontent.com/30431079/125089108-bf2db900-e0d6-11eb-993c-0cffa61dde33.png)

Reply:
So what is the decision? Move it lower, upper or keep it as it is?

Reply:
during last demo I showed a glimpse of your work, and also @lzap mentioned we might consider to move it up since sometimes it hides the page buttons, in some pages they are even higher than what @mariaaga showed,

Maybe we can even move them to be on the top-center? But let's wait for @marisvirik design thoughts on this as well

Reply:
Example from CRHC:
<img width="701" alt="Screenshot 2021-07-13 at 10 27 51" src="https://user-images.githubusercontent.com/51960547/125418546-7fae95bf-9bfd-4f49-8925-b388a8c2eef6.png">

<img width="709" alt="Screenshot 2021-07-13 at 10 44 35" src="https://user-images.githubusercontent.com/51960547/125421437-4ff4d58f-a02c-4a33-b635-7bcc32258949.png">

No need to have a two-liner alert here. 
My suggestion based on [pf4 guidelines](https://www.patternfly.org/v4/components/alert/design-guidelines), it suffices to use a header:
"Hardware model z deleted"

Alert is smaller which solves our problem. It can be moved up, but it should still stay on the right side. 
(so no top central position) @stejskalleos @laviro

Reply:
Thanks @marisvirik!

Reply:
Maybe in this case is the problem length of the message than the headline ...

Reply:
@stejskalleos for that you should use a toast with the description under the header. That type still exists and it's used but not for simple notifications such as added/completed/deleted with no additional information.

Reply:
> That type still exists and it's used but not for simple notifications such as added/completed/deleted with no additional information.

Okay this is new information for me, we have two types of toast alerts? In PF4 there is only one: https://www.patternfly.org/v4/components/alert/

There could be some logic saying if the text is more than 80 chars put the text into body, otherwise use headline.
@laviro  What do you think?

Reply:
@stejskalleos  my bad, I'm talking about variations 
<img width="860" alt="Screenshot 2021-07-13 at 15 09 46" src="https://user-images.githubusercontent.com/51960547/125457429-5ce4c9fe-40a7-4281-ab60-b80036cb181b.png">
 
 and here are some more tips about writing notifications (citation from Alana from the content team) based on PatternFly guidance for alert content and error messages.
 

- Avoid restating the same information from the alert title in the description. If the alert title communicates everything users need to know, we can remove the description altogether.

- When it comes to writing error messages and error alerts, we can write all verbs in the simple past tense to keep strings concise: We can simplify complex verb structures like “has been deleted” to “deleted.”

- Avoid using extra words like “successfully.”

- Never punctuate alert titles. Odds are, if a title forms a full sentence, it can be trimmed down to its bare essentials.

As for including item names in alert titles, we typically indicate named items by bolding or using double quotation marks to align with US grammar standards (e.g. in already bolded titles).

The alert titles should concisely communicate the alert’s main message.